deploy/docker/Dockerfile

```diff
@@ -1,8 +1,4 @@
-FROM python:3.8.7-slim
-
-# Update
-RUN apt update -y && apt upgrade -y
-RUN apt install patch gcc libffi-dev wget -y
+FROM kprasch/rust-python:3.8.9
```

Comment:
Can we host this in a nucypher Docker repo?

Reply:
Yes.  I'll push those containers to the nucypher docker repo and make a follow-up PR.